sdcm/tester.py

```diff
@@ -2611,7 +2611,7 @@ def check_regression(self):
                                                       send_email=self.params.get('send_email'),
                                                       email_recipients=self.params.get('email_recipients'),
                                                       events=get_events_grouped_by_category(
-                                                          _registry=self.events_processes_registry))
+                                                          _registry=self.events_processes_registry, limit=10))
```

Comment:
i would set maybe a variable for it to be able to change in 1 single place, but not mandatory

Reply:
yes, probably a good idea.

Reply:
WUT about new config parameter `perf_events_limit_in_email`. by default it will be 10, and could be configure per yaml for performance test?
@fgelcer @roydahan

Reply:
that would be nice, but could be called only `events_limit_in_email` for general purpose usage.

Reply:
Done